src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryUnary.cs

```diff
@@ -335,6 +341,30 @@ public static FormulaValue TextToBoolean(IRContext irContext, StringValue[] args
             return CommonErrors.InvalidBooleanFormatError(irContext);
         }
 
+        public static FormulaValue GetBoolean(IRContext irContext, FormulaValue value)
```

Comment:
>FormulaValue [](http://example.com/codeflow?start=22&length=12)

Nit - can this be stronger typed, BooleanValue

Reply:
I see this was following the existing pattern - the difference is that:
- for returning errors back to *power fx expressions*, we use ErrorValue (which CommonErrors produces). These can be caught and handled within the expression. 
- but for returning errors back to the host, we either throw exceptions (if it's a host bug) , or return false in a TryGet pattern where host can handle it. 

Since this is taking a runtime function and splitting it into 2 layers (runtime and a host ) , there's a bit of a change.